vendor-extra/ViewsBundle/src/Views/StringViewConverter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ViewsBundle\Views;
+
+use FluentDOM\DOM\CdataSection;
+use FluentDOM\DOM\Element;
+use FluentDOM\DOM\Node\NonDocumentTypeChildNode;
+use FluentDOM\DOM\Text;
+
+final class StringViewConverter implements ViewConverter
```

Comment:
I guess this is a key part

Reply:
There is also `EmptyViewConverter` that produces `View` objects that do not refer to a template; but it's only used in tests.